stdlib/tkinter/__init__.pyi

```diff
@@ -179,7 +179,7 @@ _Compound: TypeAlias = Literal["top", "left", "center", "right", "bottom", "none
 _Cursor: TypeAlias = str | tuple[str] | tuple[str, str] | tuple[str, str, str] | tuple[str, str, str, str]
 # example when it's sequence:  entry['invalidcommand'] = [entry.register(print), '%P']
 _EntryValidateCommand: TypeAlias = str | list[str] | tuple[str, ...] | Callable[[], bool]
-_GridIndex: TypeAlias = int | str
+_GridIndex: TypeAlias = int | str | Iterable[int]
```

Comment:
As the name suggests, `_GridIndex` represents one index, not a sequence of indexes. Instead, you should change `grid_rowconfigure()` and `grid_columnconfigure()`, because it's just those two methods that accept lists and tuples. I explained this in more detail on #9784.

Also, my "only works with lists and tuples" comment in your other PR applies here too.